clarity/contracts/dcards-nft.clar

```diff
@@ -2,9 +2,6 @@
 ;; NFT contract for stacks cards
 ;; https://cards.layerc.xyz
 
-
-;; (impl-trait .nft-trait.nft-trait)
```

Comment:
i left this as a reference, also for testing sometimes uncomment it since it can use the local NFT?
it's just a hassle to remember to change before each deploy based on the target network...